### ISSUE TITLE
XRT build error - protobuf generated files version mismatch

### ISSUE THREAD

**Issue by rudolpjo** [Xilinx/XRT#4792]
I have a more thorough explanation of the issue on the Xilinx forum here: https://forums.xilinx.com/t5/AI-and-Vitis-AI/Xilinx-Run-Time-XRT-build-script-error-protobuf-version/td-p/1210669

The following errors after the initial one mentioned in the post are all related to the protobuf library in one way on another.

**Comment by rudolpjo** [comment]
Can someone please take a look at this?

**Comment by sonals** [comment]
XRT uses native libraries that ship with the Linux distribution. Can you use native protobuf that comes with Ubuntu? Install the following packages with **apt install** and remove your own build of protobuf:

```
libprotoc-dev 
protobuf-compiler
```

**Comment by rudolpjo** [comment]
Thank you for responding to this issue, I tried what you suggested but got the same errors. Interestingly enough, after I removed the protobuf files from `/usr/include/google` and ran apt install, nothing was downloaded or updated. Got the message that all was up to date. Perhaps I am missing something with the location of the needed dependencies. 
When the issue first occurred, I was using the native Ubuntu 18.04 libraries, and there was a protobuf error regarding a missing file. I have tried searching for what version of protobuf ships with Ubuntu 18.04, but I have not been able to find that information out.
